src/sidebar/store/modules/test/selection-test.js

```diff
@@ -281,12 +281,12 @@ describe('sidebar/store/modules/selection', () => {
       );
       assert.equal(store.focusModeUserPrettyName(), 'FakeDisplayName');
     });
-    it('should the username when displayName is missing', function() {
+    it('should the userid when displayName is missing', function() {
```

Comment:
```suggestion
    it('returns the userid when displayName is missing', function() {
```

This was an existing typo